models.py

```
@@ -28,60 +27,79 @@
         if not self.is_running:
-            os.environ["MM_CONF_PATH"] = self.mm2_config
-            if not os.path.isfile(MM2BIN):
-                error_print(f"mm2 binary not found in {MM2BIN}!")
-                get_mm2()
-            mm2_output = open(self.mm2_log, "w+")
+            os.environ["MM_CONF_PATH"] = self.kdf_config
+            if not os.path.isfile(KDFBIN):
+                error_print(f"kdf binary not found in {KDFBIN}!")
+                get_kdf()
+            kdf_output = open(self.kdf_log, "w+")
             subprocess.Popen(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)